.github/workflows/ping-assignees.yml

```diff
@@ -0,0 +1,31 @@
+name: Ping Assignees
+
+on:
+  # Every day at 0:00
+  schedule: 0 0 * * *
+
+jobs:
+  ping_assignees:
+    name: Ping Assignees
+    runs-on: ubuntu-latest
+    steps:
+      - name: Ping Assignees
+        uses: babel/actions/check-assigned-issues@v2
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+          bot_name: babel-bot
+          days_before_warning: 14
+          days_before_unassigning: 70
```

Comment:
If we leave 7, I think that this action will unassign issues inactive for 3 weeks (2 + 1) without first giving the warning. In one week I will decrease it back to 7.